SparseGrids/tsgLinearSolvers.cpp

```diff
@@ -31,6 +31,7 @@
 #ifndef __TASMANIAN_LINEAR_SOLVERS_CPP
 #define __TASMANIAN_LINEAR_SOLVERS_CPP
 
+#include <cassert>
```

Comment:
Put this line in `tsgEnumerate.hpp`, that is a top level header and includes all the C++ STL heades that we need.

Reply:
Done.